.gitignore

```diff
@@ -0,0 +1,10 @@
+**/*.in
```

Comment:
Usually `.in` files are use for generating some other files. I don't think we should ignore them.

Reply:
These files are autogenerated from `Makefile.am` files. They should not be commited in a repository.

Reply:
```
$ find . -name "*.in"
./ta-lib.dpkg.in
./ta-lib.pc.in
./ta-lib.spec.in
```

What about these files?

Reply:
Fixed (and improved, now ignoring all generated files).